config/prod.exs

```diff
@@ -17,6 +17,7 @@ config :ex_backend, ExBackendWeb.Endpoint,
   load_from_system_env: true,
   url: [host: "backend.media-io.com", port: 443],
   root: ".",
+  check_origin: false,
```

Comment:
Please revert it, it's for your custom temporary deployment.
You've to support HTTPS and use the right domain name.

Reply:
Exact.